typescript/ecs/cluster/index.ts

```diff
@@ -11,20 +10,25 @@ class ECSCluster extends cdk.Stack {
     const vpc = new ec2.Vpc(this, 'MyVpc', { maxAZs: 2 });
 
     const asg = new autoscaling.AutoScalingGroup(this, 'MyFleet', {
-      instanceType: new InstanceType("t2.xlarge"),
+      instanceType: new ec2.InstanceType(ec2.InstanceSize.MICRO),
       machineImage: new ecs.EcsOptimizedAmi(),
-      updateType: autoscaling.UpdateType.ReplacingUpdate,
+      updateType: autoscaling.UpdateType.REPLACING_UPDATE,
       desiredCapacity: 3,
-      vpc
+      vpc,
     });
 
     const cluster = new ecs.Cluster(this, 'EcsCluster', { vpc });
     cluster.addAutoScalingGroup(asg);
 
-      cluster.addCapacity('DefaultAutoScalingGroup', {
```

Comment:
Can you create an issue in https://github.com/awslabs/aws-cdk to track this issue?

Reply:
Leave the code in while the issue is being tracked. While the issue is being investigated, the commented code obfuscates the problem

Reply:
@pkandasamy91 @NGL321 Ok I can leave the code but it won't pass the travis build is this ok?

Reply:
4a200026d143157713619498df8c92e70edd316d